docs/resources/agent_to_agent.md

```diff
@@ -165,59 +165,52 @@ Optional:
 - `network` (String) The name of the autonomous system in which the BGP monitor is found.
 
 
-<a id="nestedblock--groups"></a>
-### Nested Schema for `groups`
+<a id="nestedblock--shared_with_accounts"></a>
+### Nested Schema for `shared_with_accounts`
 
 Required:
 
-- `group_id` (Number) The unique ID of the label.
-
-Optional:
-
-- `agents` (Block List) Define the ThousandEyes agents to use. (see [below for nested schema](#nestedblock--groups--agents))
-- `name` (String) The name of the label.
-- `tests` (Block List) The list of tests. (see [below for nested schema](#nestedblock--groups--tests))
-- `type` (String) [tests, agents, endpoint_tests, or endpoint_agents] The type of label.
+- `aid` (Number) The account group ID.
 
 Read-Only:
 
-- `builtin` (Boolean) Shows whether you are using built-in (true) labels or user-created (false) labels. Built-in labels are read-only.
-
-<a id="nestedblock--groups--agents"></a>
-### Nested Schema for `groups.agents`
-
-Optional:
-
-- `agent_id` (Number) The unique ThousandEyes agent ID.
+- `name` (String) The name of account.
```

Comment:
"Name of the account"

Reply:
Thanks, these docs were written long before we got this project, so I actually haven't reviewed them to check the grammar